icu4c/source/i18n/measunit_impl.h

```diff
@@ -42,6 +42,12 @@ struct U_I18N_API SingleUnitImpl : public UMemory {
      */
     const char *getSimpleUnitID() const;
 
+    /**
+     * Generates and append a neutral identifier string for a single unit which means we do not include
+     * the dimension signal.
```

Comment:
What is a "dimension signal"? This sentence doesn't clarify much for me - I don't know what it's appending without reading the code, and I also need to notice that it's appended to the first parameter (not to *this).

In the code I see this:
int32_t absPower = std::abs(this->dimensionality);
I see dimensionality _is_ included in the output - it's only whether the dimensionality is positive or negative that's skipped - i.e. the sign / "signum" is skipped.